packages/manager/modules/nutanix/src/dashboard/component/order-private-bandwidth/order-private-bandwidth.html

```diff
@@ -0,0 +1,127 @@
+                            data-model="$ctrl.model.plan"
+                            data-value="plan.planCode"
+                        >
+                            <span
+                                data-translate="cluster_vrack-bandwidth"
+                                data-translate-values="{'bandwidth' : plan.bandwidth / 1000}"
+                            ></span>
+                        </oui-radio>
+                    </div>
+                    <div class="col-md-3 pl-0">
+                        <ovh-manager-catalog-price
+                            data-price="plan.prices[2].priceInUcents"
+                            data-tax="plan.prices[2].tax"
+                            data-interval-unit="{{plan.intervalUnit}}"
+                            data-user="$ctrl.user"
+                        >
+                        </ovh-manager-catalog-price>
+                    </div>
+                </div>
+            </div>
+
+            <oui-message class="mb-3" data-type="info">
+                <span data-translate="cluster_order_bandwidth_info"></span>
+            </oui-message>
+
+            <oui-message data-type="warning">
+                <span data-translate="cluster_order_bandwidth_warning"></span>
+            </oui-message>
+        </form>
+    </div>
+    <div
+        data-wizard-step
+        data-wizard-step-valid="$ctrl.steps[1].isValid()"
+        data-wizard-step-on-load="$ctrl.initSecondStep.bind($ctrl)"
+    >
+        <div class="text-center" data-ng-if="$ctrl.steps[1].isLoading()">
+            <oui-spinner></oui-spinner>
+            <span
+                data-translate="cluster_order_bandwidth_vrack_load"
+                class="d-block"
+            ></span>
+        </div>
+        <form data-ng-if="!$ctrl.steps[1].isLoading()">
+            <p data-translate="select_bandwidth_plan_summary"></p>
+            <div class="row">
+                <div class="col-md-8 pr-0">
+                    <oui-radio
+                        data-name="bandwidth-plan"
+                        data-model="$ctrl.model.plan"
+                        data-value="$ctrl.provisionalPlan.planCode"
+                    >
+                        <span
+                            data-translate="cluster_vrack-bandwidth"
+                            data-translate-values="{ bandwidth: $ctrl.provisionalPlan.bandwidth / 1000 }"
+                        ></span>
+                    </oui-radio>
+                </div>
+                <div class="col-md-4 pl-0">
+                    <span
+                        data-ng-bind-html="$ctrl.provisionalPlan.order.prices | ducPrice: $ctrl.user.ovhSubsidiary:'verbose'"
+                    ></span>
+                </div>
+            </div>
+            <oui-checkbox
+                data-model="$ctrl.model.autoPayWithPreferredPaymentMethod"
+                class="mt-3"
+                data-ng-if="$ctrl.region !== 'US' && $ctrl.hasDefaultPaymentMethod"
```

Comment:
May be we can use this:
```suggestion
                data-ng-if="$ctrl.coreConfig.isRegion('US') && $ctrl.hasDefaultPaymentMethod"
```